package.json

```diff
@@ -53,19 +53,19 @@
     "vinyl-source-stream": "^1.1.0"
   },
   "browser": {
-    "./lib/server/AjaxCallsForCurrentRequest.js": "./lib/util/noop.js",
-    "./lib/server/DomainLocalStorage.js": "./lib/util/noop.js",
-    "./lib/server/Server.js": "./lib/util/noop.js",
-    "./lib/server/ServerApp.js": "./lib/util/noop.js",
-    "./lib/server/ServerConfigure.js": "./lib/util/noop.js",
-    "./lib/server/ServerDispatcher.js": "./lib/util/noop.js",
-    "./lib/server/serverJquery.js": "./lib/util/noop.js",
-    "./lib/server/ServerRenderer.js": "./lib/util/noop.js",
-    "./lib/server/ServerRenderingWorkflow.js": "./lib/util/noop.js",
-    "./lib/server/ServerRequest.js": "./lib/util/noop.js",
-    "./lib/server/ServerResponse.js": "./lib/util/noop.js",
-    "./lib/server/ServerResponseWorkflow.js": "./lib/util/noop.js",
-    "./lib/server/ServerRouter.js": "./lib/util/noop.js"
+    "./lib/server/AjaxCallsForCurrentRequest.js": false,
```

Comment:
updating all of these to false per the spec https://github.com/defunctzombie/package-browser-field-spec